bumble/hci.py

```diff
@@ -1491,7 +1491,7 @@ def dict_from_bytes(data, offset, fields):
             elif field_type == -2:
                 # 16-bit signed
                 field_value = struct.unpack_from('<h', data, offset)[0]
-                offset += 1
+                offset += 2
```

Comment:
good catch! I dont think any of the commands use that field type yet, but that's a bug nonetheless.